string color_name
---
bool success
int16[] min
int16[] max